package.json

```diff
@@ -1,174 +1,104 @@
     "start": "ng serve",
     "build": "ng build --build-optimizer --delete-output-path --optimization --progress --vendor-chunk --aot",
-    "build:prod": "npm run ng build -- --prod --configuration=deploy",
-    "test": "jest --coverage",
-    "test:dev": "jest --watch",
-    "lint": "ng lint",
-    "lint:fix": "ng lint $npm_package_name --fix",
-    "e2e": "cypress open",
+    "watch": "ng build --watch",
     "sonar-scanner": "npm run test && sonar-scanner",
-    "test:deploy": "npm run build && npx http-server ./dist -p 4200 -c-1",
-    "prepush": "npm run build && npm test",
-    "security-checks": "git secrets --scan && git log -p | scanrepo",
-    "coverage": "jest --coverage",
-    "audit-dep": "npm audit --audit-level=high --json | audit-filter --nsp-config=.nsprc --audit=-"
+    "security-checks": "git secrets --scan",
+    "lint": "npx eslint 'src/**/*.{js,jsx,ts,tsx,html,css,scss}'",
+    "format": "npx prettier 'src/**/*.{js,jsx,ts,tsx,html,css,scss}' --write",
+    "docs:json": "compodoc -p ./tsconfig.json -e json -d .",
+    "storybook": "npm run docs:json && start-storybook -p 6006",
+    "build-storybook": "npm run docs:json && build-storybook",
+    "test": "jest --maxWorkers=3 --colors --coverage",
+    "test:watch": "jest --watch",
+    "test:ci": "jest -ci --runInBand --watchAll=false --reporters=default",
+    "mock-api": "npx ts-node ./mock-api/mock-api-server.ts",
+    "prepare": "husky install && ts-patch install -s"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "6.1.0-rc.3",
-    "@angular/cdk": "6.4.0",
-    "@angular/common": "6.1.0-rc.3",
-    "@angular/compiler": "6.1.0-rc.3",
-    "@angular/core": "6.1.0-rc.3",
-    "@angular/forms": "6.1.0-rc.3",
-    "@angular/http": "6.1.0-rc.3",
-    "@angular/material": "6.4.0",
-    "@angular/platform-browser": "6.1.0-rc.3",
-    "@angular/platform-browser-dynamic": "6.1.0-rc.3",
-    "@angular/router": "6.1.0-rc.3",
-    "@azure/msal-angular": "^0.1.4",
-    "@fortawesome/angular-fontawesome": "^0.3.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.25",
-    "@fortawesome/free-brands-svg-icons": "^5.11.2",
-    "@fortawesome/free-regular-svg-icons": "^5.11.2",
-    "@fortawesome/free-solid-svg-icons": "^5.11.2",
-    "@ngrx/effects": "6.0.1",
-    "@ngrx/router-store": "6.0.1",
-    "@ngrx/store": "6.0.1",
-    "@ngrx/store-devtools": "6.0.1",
-    "core-js": "^2.6.11",
-    "faker": "^4.1.0",
-    "file-saver": "^2.0.2",
-    "govuk-frontend": "^3.3.0",
-    "hammerjs": "^2.0.8",
-    "husky": "^3.0.9",
-    "install": "^0.13.0",
-    "jasmine-core": "^2.8.0",
-    "jasmine-marbles": "^0.4.0",
-    "jest-preset-angular": "^7.1.1",
-    "microsoft-adal-angular6": "^1.3.0",
-    "ngrx-forms": "3.2.1",
-    "ngrx-store-freeze": "^0.2.4",
-    "ngx-filesaver": "^2.2.1",
-    "ngx-json-viewer": "^2.4.0",
-    "rxjs": "6.2.2",
-    "rxjs-compat": "^6.5.3",
-    "uuid": "^7.0.2",
-    "zone.js": "0.8.25"
+    "@angular/animations": "~13.2.0",
```

Comment:
Normalise all versions to either tilde, caret or fixed.

Reply:
Normalised to ^